mmv1/third_party/terraform/website/docs/d/apphub_discovered_workload.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+subcategory: "Apphub"
+description: |-
+  Get information about a discovered workload.
+---
+
+# google\_apphub\_discovered_workload
+
+Get information about a discovered workload from its uri.
+
+
+## Example Usage
+
+
+```hcl
+data "google_apphub_discovered_workload" "my-workload" {
+  location = "my-location"
+  workload_uri = "my-workload-uri"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - The project where we are looking up the discovered workload from
```

Comment:
The project where the discovered workload is looked up from

Reply:
Done.